### PR TITLE
Change title to Kiali from default Patternfly

### PR DESCRIPTION
**Pull request by mwringe**

### CODE REVIEW

**Comment by mwringe** [comment]
From:
screenshot from 2018-03-14 18-28-02

To:
screenshot from 2018-03-14 18-27-59

**Comment by mtho11** [comment]
Long live Kiali!

File: src/components/Nav/Navigation.tsx [hunk 1]
```diff
@@ -14,7 +14,7 @@ const servicesPath = '/services';
 const servicesTitle = 'Services';
 
 const pfLogo = require('../../img/logo-alt.svg');
```
Comment (mtho11) [inline src/components/Nav/Navigation.tsx]: The patternfly logo can also be removed.

File: src/components/Nav/Navigation.tsx [hunk 2]
```diff
@@ -14,7 +14,7 @@ const servicesPath = '/services';
...
-const pfBrand = require('../../img/brand-alt.svg');
```
Comment (mtho11) [inline src/components/Nav/Navigation.tsx]: We can probably delete the `img/brand-alt.svg` from patternfly now too.

**Comment by mtho11** [comment]
Also need to change:  <VerticalNav.Masthead title="Swift Sunshine"> in https://github.com/mtho11/swsui/blob/540ff78435bed4b7a1f410ee3565d0899158bdf6/src/components/Nav/Navigation.tsx#L41

**Comment by josejulio** [comment]
If doing these changes in the scope of this PR. could also change this one: https://github.com/mtho11/swsui/blob/540ff78435bed4b7a1f410ee3565d0899158bdf6/src/components/About/AboutUIModal.tsx#L71

**Pull request merged**